docs/data/base/components/select/select.md

```diff
@@ -93,10 +94,50 @@ For the sake of brevity, the rest of the demos throughout this doc will not use
 ### Multi-select
 
 The Select component lets your users select multiple options from the list.
+In constrast to the single-selection mode, the options popup does not close after an item is selected, allowing users to continue choosing mor e options.
+
 Set the `multiple` prop to turn on the multi-selection mode.
 
 {{"demo": "UnstyledSelectMultiple.js", "defaultCodeOpen": false}}
 
+Note that in the multiple selection mode, the `value` prop (and `defaultValue`) is an array.
+
+### Controlled select
+
+Select can be used as an uncontrolled or controlled component:
```

Comment:
We we've learned with feedback in the X docs, we should add a callout here to briefly explain this concept. We could copy the text from this PR: https://github.com/mui/mui-x/pull/8926